src/App.tsx

```diff
@@ -10,33 +12,25 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setselectedTab] = useState(tabs[0]);
+
+  const onTabSelected = (tab:Tab) => {
+    setselectedTab(tab);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {
+          `Selected tab is ${selectedTab.title}`
+        }
```

Comment:
You can fit it in one line
```suggestion
        {`Selected tab is ${selectedTab.title}`}
```